avogadro/qtplugins/customelements/customelements.cpp

```diff
@@ -46,12 +59,12 @@ QString CustomElements::description() const
 
 QList<QAction*> CustomElements::actions() const
 {
-  return QList<QAction*>() << m_reassignAction;
+  return QList<QAction*>() << m_reassignUsingTool << m_reassignFromFile;
 }
 
 QStringList CustomElements::menuPath(QAction*) const
 {
-  return QStringList() << tr("&Build");
+  return QStringList() << tr("&Build") << tr("Assign &Custom Elements...");
```

Comment:
I'm unsure on naming here. Generally I hear "topology" more - I'm not sure if one is more prevalent. What do you think?